src/modules/jcms_migrate/src/Plugin/migrate/process/JCMSSplitContent.php

```diff
@@ -253,6 +243,7 @@ public function DOMnodeInnerHTML($element) {
       foreach ($element as $child) {
         $html = $child->ownerDocument->saveHTML($child);
         $html = str_replace('&nbsp;', ' ', $html);
+        $html = check_markup($html, 'basic_html');
```

Comment:
@nlisgo There are a couple of ways you can do it: https://www.drupal.org/docs/8/phpunit/unit-testing-more-complicated-drupal-classes